src/modules/swap/ModuleSwap.vue

```diff
@@ -693,9 +696,13 @@ export default {
      * to swap from
      */
     fromTokens() {
-      return this.availableTokens.fromTokens.map(token => {
-        return localContractToToken[token.contract];
-      });
+      return this.availableTokens.fromTokens
+        .map(token => {
+          return localContractToToken[token.contract];
+        })
+        .filter(
```

Comment:
This filter should be in the initial setup tokens since this only needs to happen once.